src/cares_wrap.cc

```diff
@@ -1990,8 +1991,8 @@ void GetAddrInfo(const FunctionCallbackInfo<Value>& args) {
                                *hostname,
                                nullptr,
                                &hints);
-  if (err)
-    delete req_wrap;
+  if (err == 0)
+    req_wrap.release();
```

Comment:
Please use `USE()` and add a comment that this indicates passing ownership to `req`

Reply:
Sorry, I missed this one. I'll update shortly. Thanks